coordinator/internal/seedengine/seedengine.go

```diff
@@ -82,15 +79,12 @@ func New(secretSeed []byte, salt []byte) (*SeedEngine, error) {
 	return se, nil
 }
 
-// DerivePodSecret derives a secret for a pod from the policy hash and the secret seed.
-func (s *SeedEngine) DerivePodSecret(policyHash [hashSize]byte) ([]byte, error) {
-	if policyHash == [hashSize]byte{} {
-		return nil, errors.New("policy hash must not be empty")
-	}
-	if bytes.Equal(policyHash[:], s.hashFun().Sum(nil)) {
-		return nil, errors.New("policy hash is the hash of an empty byte slice")
+// DeriveWorkloadSecret derives a secret for a workload from the workload name and the secret seed.
+func (s *SeedEngine) DeriveWorkloadSecret(workloadSecretID string) ([]byte, error) {
+	if workloadSecretID == "" {
+		return nil, errors.New("workload name must not be empty")
 	}
-	return s.hkdfDerive(s.podStateSeed, fmt.Sprintf("POD SECRET %x", policyHash))
+	return s.hkdfDerive(s.podStateSeed, fmt.Sprintf("workload-key:%s", workloadSecretID))
```

Comment:
I first wrote the extra function after the RFC https://github.com/edgelesssys/contrast/blob/main/rfc/007-workload-secrets.md?plain=1#L88 and then removed the old one, so I didn't change the original function in that sense. 
Since I don't see a technical argument, I'll align with the package.